src/pages/CreepypastaPage.js

```diff
@@ -1,27 +1,45 @@
 import { NavLink } from 'react-router-dom';
 import styled from 'styled-components';
 import Card from '../components/Card.js';
+import Searchbar from '../components/Searchbar.js';
 import headerPicture from '../Images/Header.png';
 
-export default function CreepypastaPage({ handleBookmarkClick, creepypastas }) {
+export default function CreepypastaPage({
+  handleBookmarkClick,
+  creepypastas,
+  handleChange,
+  searchValue,
+}) {
   return (
     <>
       <Link to="/spookmarked">Spookmarked</Link>
       <Header>
         <img alt="CREEPY(pen)NE" src={headerPicture} width="300" />
       </Header>
       <Grid>
-        {creepypastas.map(creepypasta => (
-          <Card
-            key={creepypasta.id}
-            id={creepypasta.id}
-            title={creepypasta.title}
-            image={creepypasta.image}
-            wordcount={creepypasta.wordcount}
-            onBookmarkClick={() => handleBookmarkClick(creepypasta.id)}
-            isBookmarked={creepypasta.isBookmarked}
-          />
-        ))}
+        <Searchbar handleChange={handleChange} searchValue={searchValue} />
+        {creepypastas &&
+          creepypastas
+            .filter(creepypasta =>
+              creepypasta.title.trim().toLowerCase().includes(searchValue)
```

Comment:
Cause you are using these lines below again to check for the error, you could save the filtered results probably in a variable and use the variable to map over the results and to check the length to display the error message.